asm_utils/src/utils.rs

```diff
@@ -1,7 +1,15 @@
 use crate::ast::{Argument, Expression, FunctionOpKind, Register};
 
-pub fn next_multiple_of_four(x: usize) -> usize {
-    ((x + 3) / 4) * 4
+pub fn next_aligned(val: usize, alignment: usize) -> usize {
+    // Alignment will probably always be a power of two, which can be aligned in
+    // a much faster bitwise operation. But then we would have to assert!() it,
+    // so it is just better to use the generic version.
+    ((val + (alignment - 1)) / alignment) * alignment
+}
+
+pub fn alignment_size(from: usize, alignment: usize) -> usize {
```

Comment:
Can you document this? Can the be described as "padding to next alignment boundary"?

Reply:
Done.